openshift/system/Dockerfile

```diff
@@ -29,19 +30,19 @@ RUN sed -i 's/mirrorlist/#mirrorlist/g' /etc/yum.repos.d/CentOS-* \
 # can just install memkind once it is fixed, see
 # https://issues.redhat.com/browse/RHEL-14497
 RUN cd /tmp \
-    && dnf download --source memkind \
+    && dnf download http://mirror.stream.centos.org/9-stream/AppStream/source/tree/Packages/memkind-1.11.0-2.el9.src.rpm \
```

Comment:
For some reason couldn't download this with `dnf download --source memkind` :shrug:

Reply:
That's weird, what's the error?

Reply:
Hm... I didn't keep the log, but it was saying something like "not found"

Now it indeed works on the base image, so I'll try again.